### PR TITLE
Unwrap eth on sell

### PR DESCRIPTION
**Pull request by alfetopito**
# Summary

Using special ETH address (`0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeee`) when buying ETH

Sample order request payload:

```
{"sellToken":"0xc7AD46e0b8a400Bb3C915120d284AafbA8fc4735","buyToken":"0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE","sellAmount":"100000000000000000000000000","buyAmount":"524233710758060","validTo":1610404372,"appData":0,"feeAmount":"0","kind":"sell","partiallyFillable":false,"signature":"0x14a0e773aa004f43307173280a810d1c42008961cc63c720af679006f3c0c8cd1fbd14907a48e2b4cc356257267d3aeacc92dbc1040432a63995775cebdd7f281c"}
```

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
   * **🎩 [Dapp](https://pr96--gpswapui.review.gnosisdev.com)**: Testing web app

File: src/custom/constants/index.ts [hunk 1]
```diff
@@ -18,3 +18,14 @@ export const GP_ALLOWANCE_MANAGER_CONTRACT_ADDRESS: Partial<Record<ChainId, stri
   [ChainId.RINKEBY]: GPv2AllowanceManager[ChainId.RINKEBY].address
   // [ChainId.xDAI]: GPv2AllowanceManager[ChainId.xDAI].address // Not yet deployed
 }
+
+// See https://github.com/gnosis/gp-v2-contracts/commit/821b5a8da213297b0f7f1d8b17c893c5627020af#diff-12bbbe13cd5cf42d639e34a39d8795021ba40d3ee1e1a8282df652eb161a11d6R13
+export const BUY_ETHER_ADDRESS = '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE'
```
Comment (W3stside) [inline src/custom/constants/index.ts:23]: rEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE

**Review by W3stside — APPROVED** [review]
looking good

File: src/custom/constants/index.ts (hunk 1 above)
Comment (alfetopito) [reply src/custom/constants/index.ts:23]: eeeeeeee

**Review by anxolin — APPROVED** [review]
👌

**Pull request merged**